apps/openchallenges/challenge-service/src/main/resources/openapi.yaml

```diff
@@ -710,6 +712,9 @@ components:
         websiteUrl: https://openchallenges.io
         name: name
         id: 1
+        categories:
+        - featured
+        - featured
```

Comment:
Double featured?

Reply:
The file `[apps/openchallenges/challenge-service/src/main/resources/openapi.yaml](https://github.com/Sage-Bionetworks/sage-monorepo/pull/2404/files/2e7a5ac59f7656aacc00a19ee319bdff6ef65694#diff-76fa348d73b49d4f8047ca61c668bcee8289016590ae155948494163883fd457)` is automatically generated so we don't want to modify it manually.

Reply:
Whoa, nice catch.  Not sure how this happened, as `nx build` was used to generate the API spec.  Will try to re-generate to see if this gets fixed.

Reply:
It's the example that OpenAPI automatically generate. We could write our own example but that's a detail.

Reply:
🙃 

![Screenshot 2024-02-13 at 9 43 25 AM](https://github.com/Sage-Bionetworks/sage-monorepo/assets/9377970/e0dead80-c957-4a4f-b583-429e25516fc4)

Reply:
I'm wondering if this is perhaps the result of a merging conflict went awry 🙃